server/planning/assignments/assignments_complete.py

```diff
@@ -66,7 +66,8 @@ def on_update(self, updates, original):
                     raise SuperdeskApiError.forbiddenError('Update Assignment not in correct state.')
             elif not original.get('scheduled_update_id') and assignment_state != ASSIGNMENT_WORKFLOW_STATE.IN_PROGRESS:
                 raise SuperdeskApiError.forbiddenError('Assignment not in progress.')
-        elif assignment_state not in [ASSIGNMENT_WORKFLOW_STATE.ASSIGNED, ASSIGNMENT_WORKFLOW_STATE.SUBMITTED]:
+        elif assignment_state not in [ASSIGNMENT_WORKFLOW_STATE.ASSIGNED, ASSIGNMENT_WORKFLOW_STATE.SUBMITTED,
+                                      ASSIGNMENT_WORKFLOW_STATE.IN_PROGRESS]:
             raise SuperdeskApiError.forbiddenError(
                 'Cannot confirm availability. Assignment should be assigned or submitted.')
```

Comment:
Can you update this error message to include `in progress` please

Reply:
Done